runners/spark/job-server/build.gradle

```diff
@@ -109,8 +109,7 @@ def portableValidatesRunnerTask(String name) {
       excludeCategories 'org.apache.beam.sdk.testing.UsesMapState'
```

Comment:
What about these?

Reply:
Looks like map state isn't supported yet by the fn api.
https://github.com/apache/beam/blob/master/sdks/java/harness/src/main/java/org/apache/beam/fn/harness/state/FnApiStateAccessor.java#L302

Reply:
Yep, there is only bagstate for now.